itest/rust/src/register_tests/multiple_impl_blocks_test.rs

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use godot::classes::IObject;
+use godot::obj::{Base, Gd, NewAlloc};
+use godot::register::{godot_api, GodotClass};
+
+use crate::framework::itest;
+
+// ----------------------------------------------------------------------------------------------------------------------------------------------
+
+#[derive(GodotClass)]
+#[class(base=Object)]
+struct MultipleImplBlocks {}
+
+#[godot_api]
+impl IObject for MultipleImplBlocks {
+    fn init(_base: Base<Self::Base>) -> Self {
+        Self {}
+    }
+}
+
+#[godot_api]
+impl MultipleImplBlocks {
+    #[func]
+    fn foo(&self) -> String {
+        "result of foo".to_string()
+    }
+}
+
+#[godot_api(secondary)]
+impl MultipleImplBlocks {
+    #[func]
+    fn bar(&self) -> String {
+        "result of bar".to_string()
+    }
+}
+
+#[godot_api(secondary)]
+impl MultipleImplBlocks {
+    #[func]
+    fn baz(&self) -> String {
+        "result of baz".to_string()
+    }
+}
```

Comment:
Sorry I missed this before, but please avoid names like "foo". Rationale in https://github.com/godot-rust/gdext/pull/888.

You should be able to directly commit this:
```suggestion
#[godot_api]
impl MultipleImplBlocks {
    #[func]
    fn first(&self) -> String {
        "1st result".to_string()
    }
}

#[godot_api(secondary)]
impl MultipleImplBlocks {
    #[func]
    fn second(&self) -> String {
        "2nd result".to_string()
    }
}

#[godot_api(secondary)]
impl MultipleImplBlocks {
    #[func]
    fn third(&self) -> String {
        "3rd result".to_string()
    }
}
```